CHANGELOG.md

```diff
@@ -260,10 +276,19 @@ as listed below.
 
 Initial private preview release, no longer supported.
 
-[Unreleased](https://github.com/openenclave/openenclave/compare/v0.7.0...HEAD)
+[Unreleased](https://github.com/openenclave/openenclave/compare/v0.8.0...HEAD)
```

Comment:
The syntax here should changed so that the links are in the headers:
```
[Unreleased]: https://github.com/openenclave/openenclave/compare/v0.8.0...HEAD
[v0.8.0]: https://github.com/openenclave/openenclave/compare/v0.7.0...v0.8.0)

...
```

Reply:
Sorry, I dont underst understand: so that links are in the headers? Which headers?

Reply:
See https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links. we should be using reference-style links, which is why all the headers are in braces, e.g. 
```
[Unreleased]
------------
```

Reply:
Got it. Done. Thanks @CodeMonkeyLeet